persistence-jdbc/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/slick/SlickPersistentEntityActorSpec.scala

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.scaladsl.persistence.slick
+
+import com.lightbend.lagom.scaladsl.persistence.{ AbstractPersistentEntityActorSpec, TestEntitySerializerRegistry }
+
+import scala.concurrent.ExecutionContext
+
+class SlickPersistentEntityActorSpec(implicit ec: ExecutionContext)
+  extends SlickPersistenceSpec(TestEntitySerializerRegistry)
+  with AbstractPersistentEntityActorSpec
```

Comment:
This test can be deleted, it's just testing the persistent entity actor, which is the same as for the JDBC spec, the read side isn't involved at all so it's essentially just a duplicate test.